recipes/halide/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "halide" %}
+{% set version = "2016.10.25" %}
+{% set sha256 = "2d4beac52b8c710e3001b61679eabe6d49bc5bf17abf9b284dbce81d8f5b762b" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/halide/Halide/archive/release_{{ version.replace('.', '_') }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - ldflags.patch
+
+build:
+  number: {{ build }}
+  skip: true  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - clangdev
+    - llvmdev
+    - zlib  # [linux]
```

Comment:
I added it because the build was failing on Linux without it, but OK on Mac. Now that I think about it, this should be a build and run dep on both. Fixed now.